Automattic/jetpack
# Jetpack AI: decouple modal prompt input for reusability

## Proposed changes:
This PR decouples the input we use for AI prompts on modals so we can use the same one on all modals.

### Other information:


## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
Add a logo and image block on the editor.

Use the AI generation button on the toolbar or the main block's body (image) to open the modal. See that they use the same component and look exactly the same (even if one has the styles dropdown on top and the other doesn't)

Smoke test both to make sure they keep working as expected.
